components/clm/src/main/clmfates_interfaceMod.F90

```diff
@@ -438,11 +438,19 @@ subroutine init(this, bounds_proc )
 
             ! INTERF-TODO: WE HAVE NOT FILTERED OUT FATES SITES ON INACTIVE COLUMNS.. YET
             ! NEED A RUN-TIME ROUTINE THAT CLEARS AND REWRITES THE SITE LIST
-
-            if (lun_pp%itype(l) == istsoil ) then
+            if ( lun_pp%itype(l) == istsoil ) then
                s = s + 1
                collist(s) = c
                this%f2hmap(nc)%hsites(c) = s
+
+!               if(.not.col_pp%active(c)) then
```

Comment:
@rgknox, How about deleting these commented out lines?

Reply:
doh, sorry, I should had caught that, thanks